### ISSUE TITLE
Index out of range

### ISSUE THREAD

**Issue by hcm007** [pion/webrtc#820]
panic: runtime error: index out of range
```
goroutine 102716 [running]:
github.com/pion/webrtc/v2/internal/mux.(*Mux).dispatch(0xc0313608c0, 0xc0235b2000, 0x0, 0x2000, 0x0, 0x0)
    /go/pkg/mod/github.com/pion/webrtc/v2@v2.1.3/internal/mux/mux.go:130 +0x270
github.com/pion/webrtc/v2/internal/mux.(*Mux).readLoop(0xc0313608c0)
    /go/pkg/mod/github.com/pion/webrtc/v2@v2.1.3/internal/mux/mux.go:110 +0xf6
created by github.com/pion/webrtc/v2/internal/mux.NewMux
    /go/pkg/mod/github.com/pion/webrtc/v2@v2.1.3/internal/mux/mux.go:43 +0x127
```

**Comment by hcm007** [comment]
I committed pr in this issue page `https://github.com/pion/webrtc/pull/821`

**Comment by Sean-Der** [comment]
Fixed by 0e257f6c3d8495e938f46e08bded895a9f8a03c0
